config/routes.rb

```diff
@@ -11,5 +11,5 @@
   post '/image', to: 'images#create'
   get '/500', to: 'application#server_errors'
   get '/400', to: 'application#search_errors'
-  get '(*url)', to: redirect('/404')
+  # get '(*url)', to: redirect('/404')
```

Comment:
I wonder how to get this to work eventually.

Reply:
This one line held us up all yesterday - so unfortunate